apisix/init.lua

```diff
@@ -86,6 +88,12 @@ function _M.http_init(args)
             core.log.error("failed to load the configuration: ", err)
         end
     end
+
+    local is_cli_token = getenv(cli_util.ADMIN_TOKEN_TAG_KEY)
+    if is_cli_token and tonumber(is_cli_token) == 1 then
+        core.log.warn("admin token has been automatically created by the system, ",
```

Comment:
system is too generic, using `apisix` will be better.

Reply:
updated, thx~